packages/expect/src/index.js

```diff
@@ -223,7 +224,10 @@ const makeThrowingMatcher = (
       getState(),
       {
         equals,
+        getObjectSubset,
         isNot,
+        iterableEquality,
+        subsetEquality,
```

Comment:
Thoughts on exposing these to matchers and how?

Reply:
maybe stick them in `this.utils` instead of directly on `this`?

Reply:
I had the same thought, will move

Reply:
also needs to be doced, fwiw